plugins/dql-backend/src/service/queries.ts

```diff
@@ -87,4 +88,48 @@ export const dynatraceQueries: Record<
     | fieldsRemove id, name, workload.labels, cluster.id, namespace.id
     | fieldsAdd Environment = "${apiConfig.environmentName}"`;
   },
+  [DynatraceQueryKeys.SRG_VALIDATIONS]: (entity, apiConfig) => {
+    const catalogTags =
+      entity.metadata.annotations?.['dynatrace.com/guardian-tags'];
+    let filterString = '';
+
+    if (catalogTags) {
+      const tags = catalogTags.split(',').map(tag => {
+        const [key, value] = tag.split('=');
+        return { key, value: value !== undefined ? value || '' : undefined };
+      });
+
+      if (tags.length > 0) {
+        filterString += '| filter ';
+        filterString += tags
+          .map(({ key, value }) =>
+            value !== undefined
+              ? `in (tags[\`${key}\`], "${value}")`
+              : `isNotNull(tags[${key}])`,
+          )
+          .join(' AND ');
+      }
+    }
+    return `
+    fetch bizevents, from: -2d
```

Comment:
We should also discuss how we want to proceed with the timeframe. @johannes-b  any opinions on a default timeframe for SRG validations?